library/std/src/sys/windows/c.rs

```diff
@@ -50,6 +51,9 @@ pub type CONDITION_VARIABLE = RTL_CONDITION_VARIABLE;
 pub type SRWLOCK = RTL_SRWLOCK;
 pub type INIT_ONCE = RTL_RUN_ONCE;
 
+#[cfg(target_vendor = "uwp")]
+pub const STATUS_NOT_IMPLEMENTED: NTSTATUS = 0xC0000002_u32 as _;
```

Comment:
Instead of adding this here, could you add `Windows.Win32.Foundation.STATUS_NOT_IMPLEMENTED` to [windows_sys.lst](https://github.com/rust-lang/rust/blob/d0ee1908ed791d3e91d2ad74ba502eaa203cff6d/library/std/src/sys/windows/c/windows_sys.lst#L1931). You'll also need to use `./x run generate-windows-sys` to update the code.

There really should be a larger variety of status codes in there but adding more is way beyond the scope of this PR.

Reply:
Fixed in https://github.com/rust-lang/rust/pull/112484/commits/cd523f2f184b4325b860e7705b939f7ec1cfb4f0